jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/J2735SupplementalVehicleExtensions.java

```diff
@@ -52,28 +52,28 @@ public void setVehicleData(J2735VehicleData vehicleData) {
 		this.vehicleData = vehicleData;
 	}
 
-	public J2735WeatherReport getWeatherReport() {
-		return weatherReport;
+	public J2735WeatherReport getDoNotUse1() {
+		return doNotUse1;
 	}
```

Comment:
We can use the @Data annotation to eliminate the need for explicit getters and setters in this class.

Reply:
Addressed